scripts/e2e/issue151_selection_linux.py

```
@@ -0,0 +1,201 @@
+            text=True,
+            capture_output=True,
+            timeout=0.25,
+        )
+    except subprocess.TimeoutExpired:
+        return ""
+    return result.stdout
+
+
+def set_clipboard(text: str) -> None:
+    subprocess.run(
+        ["xclip", "-selection", "clipboard", "-i"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)